src/test/java/TestPesronCreationAndData.java

```diff
@@ -0,0 +1,71 @@
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TestPesronCreationAndData {
+
+        Man person1 = new Man("Tom","Sot", 35);
+        Woman person2 = new Woman("Lola","Huston",24);
+        Man person3 = new Man("Nikki","lius",66);
+        Woman person4  = new Woman("Jane","Loverson",61);
```

Comment:
A couple things here:
1. Variable name must represent meaning so names must be like: `man1, man2, woman1, woman2`
2. Ages should have set of boundary values. For man: 64, 65, 66. For woman: 59, 60, 61